### DISCUSSION TITLE
I can't seem to purchase .is domain through Vercel

### DISCUSSION THREAD

**Discussion by mimiqkz** [vercel/community#822] [Help] [answered]
I keep getting an error `Our upstream provider is having trouble processing this request. Please try again later`.
I have been trying to buy a domain for the past 2 hours. Is there anyone that can help me? Should I try again tomorrow or am I unable to purchase .is domain through Vercel?

**Comment by swarnava** [comment, marked as answer]
Hi there, `.is` tld is indeed not supported. I have forwarded this to our product team to ensure they are blocked to purchase from our Dashboard. Note you can still purchase from different registrar and configure in Vercel: https://vercel.com/docs/concepts/projects/custom-domains
